monix-eval/shared/src/main/scala/monix/eval/CoevalLike.scala

```diff
@@ -119,7 +134,7 @@ private[eval] abstract class CoevalLikeImplicits0 {
     */
   implicit def fromComonad[F[_]](implicit F: Comonad[F]): CoevalLike[F] =
```

Comment:
Actually, do we really want this instance? Apart from `Eval`, standard comonads from Cats are things like `NonEmptyList`, and a traditional case is something like a zipper, where `extract` would lose data - not that it makes sense to convert it to `Coeval` anyway.

Reply:
OK, it probably doesn't make sense. Will just treat `Eval` as a special case and remove this.